python/ray/autoscaler/_private/fake_multi_node/bootstrap_key.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDbuAtUVt9+o4pf03LdwXwbHTYb3761ToEegfh7SXJcxZne2KJkD46ifOYssM9kw3lh14iIOmm2K5umaepSCvGvIPmBPr89t0l5gdUTnISt5ETT96/w61Zn7UjBwI58+mPGeKBdGJKlQ38Q9Mjbker2rippDXMJp8ji1Y2XZncS99y3XXz697hKIRkEwtBtJJVQyBq76qDxfe2uN9njh23pp3tNjJPFTSzXvCpq29U/oIJO/gU7Ln2iotlS77WZko6BX/w5AxFIqOkgHf9mZgLGRQ+Nh1cbypmmF/Zv7BSISbOtjT1zMPthSaW+MwD/Lr5E/JTc1z4tOMOtiuE3pbVMnsd+u2nvLvnN96weZSk25Q3Fw0HqDRFVaELG20v2/Hz+b2LW3HlCTmn9L3tKEuVZQH+LP6nPqptmTw+o0AXp3bUnM/JelVehniQ3s9vU0oXyM7g/BxDWuhbpCwTj2+XfpyZn1173tgCqwzZ/Lmm+vGMGGdRcAAjPAXXssGc6KCk= fake_multinode_docker
```

Comment:
Can we generate these instead of checking them in?

Reply:
Sure, this will require `ssh-keygen` on the host to avoid more dependencies but that shouldn't be a problem usually.